modules/react/banner/stories/stories.tsx

```diff
@@ -1,94 +0,0 @@
-/// <reference path="../../../../typings.d.ts" />
-import * as React from 'react';
-import {storiesOf} from '@storybook/react';
-import {withKnobs, select, text} from '@storybook/addon-knobs';
-import withReadme from 'storybook-readme/with-readme';
-
-import Banner from '../index';
-import README from '../README.md';
-
-const handleBannerClick = () => {
-  alert(`onClick triggered`);
-};
-
-storiesOf('Components/Indicators/Banner/React/Alert', module)
-  .addParameters({component: Banner})
-  .addDecorator(withReadme(README))
-  .addDecorator(withKnobs)
-  .add('Full', () => (
```

Comment:
Reminder that all of these story names have changed so we'll have to update the URLs on the design site.